src/plugins/unified_field_list/public/components/field_stats/field_number_summary.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { ReactNode } from 'react';
+import { EuiBasicTable, HorizontalAlignment, LEFT_ALIGNMENT, RIGHT_ALIGNMENT } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { DataView, DataViewField } from '@kbn/data-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import type { NumberSummary } from '../../../common/types';
+
+interface SummaryTableItem {
+  key: string;
+  label: ReactNode;
+  value: string;
+}
+
+export interface FieldNumberSummaryProps {
+  dataView: DataView;
+  field: DataViewField;
+  numberSummary?: NumberSummary;
+  'data-test-subj': string;
+}
+
+export const FieldNumberSummary: React.FC<FieldNumberSummaryProps> = ({
+  dataView,
+  field,
+  numberSummary,
+  'data-test-subj': dataTestSubject,
+}) => {
+  if (!numberSummary || !isNumberSummaryValid(numberSummary)) {
+    return null;
+  }
+
+  const formatter = dataView.getFormatterForField(field);
+
+  const summaryTableItems: SummaryTableItem[] = [
+    {
+      key: 'min',
+      label: (
+        <FormattedMessage
+          id="unifiedFieldList.fieldStats.numberSummary.minLabel"
+          defaultMessage="min"
+        />
+      ),
+      value: formatter.convert(numberSummary.minValue, 'text'),
+    },
+    {
+      key: 'max',
+      label: (
+        <FormattedMessage
+          id="unifiedFieldList.fieldStats.numberSummary.maxLabel"
+          defaultMessage="max"
+        />
+      ),
+      value: formatter.convert(numberSummary.maxValue, 'text'),
+    },
+  ];
+  const summaryTableColumns = [
+    {
+      name: '',
+      render: (summaryItem: { label: ReactNode }) => summaryItem.label,
+      align: LEFT_ALIGNMENT as HorizontalAlignment,
+    },
+    {
+      field: 'value',
+      name: '',
+      render: (v: string) => <strong>{v}</strong>,
+      align: RIGHT_ALIGNMENT as HorizontalAlignment,
+    },
+  ];
+
+  const summaryTableTitle = i18n.translate(
+    'unifiedFieldList.fieldStats.numberSummary.summaryTableTitle',
+    {
+      defaultMessage: 'Summary',
+    }
+  );
+
+  return (
+    <EuiBasicTable<SummaryTableItem>
+      compressed
+      items={summaryTableItems}
+      columns={summaryTableColumns}
+      tableCaption={summaryTableTitle}
+      data-test-subj={`${dataTestSubject}-numberSummary`}
+      responsive={false}
+      css={css`
+        & .euiTableHeaderCell {
+          display: none;
+        }
```

Comment:
```suggestion
        & thead {
          display: none;
        }
```
For a11y reasons, I think we should hide the entire `thead` since currently a screen reader will say there are 3 rows instead of 2 since it's still counting the `thead > tr`.

An alternative could be to fill out the `name` props in `summaryTableColumns` (could use "Label" and "Value") and use `euiScreenReaderOnly()` on the header cells:
```
& .euiTableHeaderCell {
  ${euiScreenReaderOnly()}
}
```

Either one would be fine in my opinion.